parsec/interfaces/dtd/insert_function.c

```diff
@@ -2273,11 +2444,14 @@ static void
 parsec_dtd_destroy_task_class(parsec_dtd_taskpool_t *dtd_tp, parsec_task_class_t *tc)
 {
     parsec_dtd_task_class_t *dtd_tc = (parsec_dtd_task_class_t *)tc;
+    __parsec_chore_t **incarnations;
     int j, k;
 
     /* We only register CPU hooks in the functions hash table */
     uint64_t fkey = (uint64_t)dtd_tc->cpu_func_ptr + tc->nb_flows;
     parsec_dtd_release_task_class(dtd_tp, fkey);
+    incarnations = (__parsec_chore_t**)tc->incarnations;
```

Comment:
Why do we need this temporary variable ? Can we just free the incarnation field ?

Reply:
Fixed in 6041fbb164df00bfeb74a355b4a202f7ff0f2ec7